src/main/java/user/createCorporatePage/PageGenerateInputBoundary.java

```diff
@@ -0,0 +1,5 @@
+//package user.createCorporatePage;
```

Comment:
Same as above, if you want to keep this input boundary class even though you're not using it anymore, you need to comment on it to explain why this extra class is still in our code.

Reply:
class deleted